rasterizer/renderer.go

```diff
@@ -1,32 +1,42 @@
-package canvas
+package rasterizer
 
 import (
 	"image"
 
+	"github.com/tdewolff/canvas"
 	"golang.org/x/image/draw"
 	"golang.org/x/image/math/f64"
 	"golang.org/x/image/vector"
 )
 
-type Rasterizer struct {
-	img draw.Image
-	dpm float64
+// Draw draws the canvas on a new image with given density (in dots-per-millimeter).
+// Higher density will result in bigger images.
+func Draw(c *canvas.Canvas, density canvas.PixelsPerMillimeter) *image.RGBA {
```

Comment:
Not sure if `density` is very clear, what about `dpm`? Otherwise perhaps `res` for resolution. We could implement a `.DPM() float64` function on density/dpm/resolution to prevent having to write `float64(density)` later on.

Reply:
The `float64(...)` syntax is already used everywhere, for instance below:
```go
return float64(size.X) / float64(r.density), float64(size.Y) / float64(r.density)
```

I tried to stick to that convention, which is also more usual than a method call:
```go
return float64(size.X) / r.res.DPM(), float64(size.Y) / r.res.DPM()
```

Moreover when this variable is used a lot, I cast it once and reuse it many times, following your convention in [`RenderPath`](https://github.com/tdewolff/canvas/pull/25/files#diff-36d60582540501d7c50b64a47807fa86R51-R55) for instance

Reply:
I just added a `MetricSize` method to the resolution type

Reply:
We could stay with `density` and use the `float64()`, that's actually fine! Not too sure if I like the new `MetricSize` method though, if it only is used in one place I prefer not to add a new method.

Reply:
I reverted the commit (via a force-push)